starport_template/lib/stores/wallets_store.dart

```diff
@@ -50,4 +65,68 @@ class WalletsStore {
     wallets.value.add(creds.publicInfo);
     return creds.publicInfo;
   }
+
+  Future getCosmosBalances(BaseEnv baseEnv, String walletAddress) async {
+    isError.value = false;
+    isBalancesLoading.value = true;
+    try {
+      final uri = '${baseEnv.baseApiUrl}/cosmos/bank/v1beta1/balances/$walletAddress';
+      final response = await http.get(Uri.parse(uri));
+      final map = response.body as Map<String, dynamic>;
+      final list = map['balances'] as List<Map<String, dynamic>>;
+      var balanceList = <BalanceJson>[];
+      if (list.isNotEmpty) {
+        balanceList = list.map((e) => BalanceJson.fromJson(e)).toList();
+      }
+
+      balancesList.value = balanceList
+          .map(
+            (e) => Balance(
+              denom: Denom(e.denom),
+              amount: Amount.fromString(e.amount),
+            ),
+          )
+          .toList();
+      isBalancesLoading.value = false;
+    } catch (ex) {
+      debugPrint(ex.toString());
+      isError.value = true;
+      isBalancesLoading.value = false;
+    }
+  }
+
+  Future sendCosmosMoney(
+    WalletPublicInfo info,
+    Balance balance,
+    String toAddress,
+  ) async {
+    isSendMoneyLoading.value = true;
+    final message = bank.MsgSend.create()
+      ..fromAddress = info.publicAddress
+      ..toAddress = toAddress;
+    message.amount.add(
+      alan.Coin.create()
+        ..denom = balance.denom.text
+        ..amount = balance.amount.value.toString(),
+    );
+
+    final unsignedTransaction = UnsignedAlanTransaction(messages: [message]);
+
+    final walletLookupKey = WalletLookupKey(
+      walletId: info.walletId,
+      chainId: info.chainId,
+      password: StarportApp.password,
+    );
+
+    final signedAlanTransaction = await _transactionSigningGateway.signTransaction(
+      transaction: unsignedTransaction,
+      walletLookupKey: walletLookupKey,
+    );
+    await signedAlanTransaction.fold<Future?>(
+      (fail) => null,
+      (signedTransaction) => _transactionSigningGateway.broadcastTransaction(
+          walletLookupKey: walletLookupKey, transaction: signedTransaction),
+    );
+    isSendMoneyLoading.value = false;
+  }
```

Comment:
this could also be a good candidate for a separate class:
```dart
TokenSender(_transactionSigningGateway).send(balance: balance, toAddress: toAddress);
```

Reply:
✅